src/pybind/mgr/volumes/module.py

```diff
@@ -223,30 +223,43 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'perm': 'r'
         },
         {
-            'cmd': 'nfs export create '
-            'name=type,type=CephString '
+            'cmd': 'nfs export create cephfs '
             'name=fsname,type=CephString '
-            'name=binding,type=CephString '
             'name=attach,type=CephString '
```

Comment:
where do you accept clusterid?

Reply:
Yes, as the exports are based on each cluster.